### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by breadxiexie** [SparkleXFantasy/MM-Det#1]
Can you provide the video dataset of training or testing? Also, you have chosen the method of reconstructing differences. Does this mean that this method is effective for most situations that may be encountered in practice? (Because many other works are also completed based on the differences in reconstruction)

**Comment by SparkleXFantasy** [comment]
Hi.
> Can you provide the video dataset of training or testing?

We have updated the link of DVF in BaiduNetDisk. The Google Driver link will be available soon.

> Does this mean that this method is effective for most situations that may be encountered in practice?

We followed previous work of [DIRE](https://arxiv.org/pdf/2303.09295) and [AEROBLADE](https://arxiv.org/pdf/2401.17879) for the reconstruction procedure. In Tab. 2 of [MM-Det](https://arxiv.org/pdf/2410.23623), we observed a slight +1.1% improvement after the reconstruction was introduced. Considering the large computational cost for the reconstruction, we think this method has limited contributions to video-level forensics tasks. In addition, [AEROBALDE](https://arxiv.org/pdf/2401.17879) observes that the effectiveness of the reconstruction is relevant to the choices of autoencoders. For detection tasks in real-world scenarios, the performance may be downgraded if the encoders of generative models are not available.

**Comment by breadxiexie** [comment]
Yes, but from my results on various test samples from the Internet, the accuracy of aeroblade's method of calculating and reconstructing the difference is the best
